src/main/java/edu/wpi/grip/core/operations/OpenCVOperations.java

```diff
@@ -0,0 +1,25 @@
+package edu.wpi.grip.core.operations;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.events.OperationAddedEvent;
+import edu.wpi.grip.core.operations.opencv.MatFieldAccessor;
+import edu.wpi.grip.core.operations.opencv.NewPointOperation;
+import edu.wpi.grip.core.operations.opencv.NewSizeOperation;
+import edu.wpi.grip.generated.CVOperations;
+
+public class OpenCVOperations extends CVOperations {
```

Comment:
This class hierarchy is a little confusing.  OpenCVOperations extends CVOperations?  It sounds like these should really be the same class, and you can just have the generated stick in the method below.